src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -95,14 +98,18 @@ export interface ActiveFIProps {
   routinePlans: Plan[] | null;
   supersetService: typeof supersetFetch;
   plan: Plan | null;
+  jurisdictionParentId: string;
+  searchedTitle: string | null;
 }
 
 /** default props for ActiveFI component */
 export const defaultActiveFIProps: ActiveFIProps = {
   caseTriggeredPlans: null,
   fetchPlansActionCreator: fetchPlans,
+  jurisdictionParentId: '',
```

Comment:
I think that `makePlansArraySelector` does not require `jurisdictionParentId` so I believe it is not necessary for us to do this.